### DISCUSSION TITLE
Sort Ranks in Tablist

### DISCUSSION THREAD

**Discussion by Luu7** [Zrips/CMI#3912] [Feature Requests]
**Description of feature/improvement request:**
_Based on permissions, you can sort the ranks in the tab. from highest to lowest, or lowest to highest._
---
### Detailed explanation of what it should do, when and in what situation: ###
_You have 3 ranks, Owner, Admin, Member. In the Tab, when you see the player, you see that the ranks are mixed.. You have in the TOP a Member player and in the Middle the Owner, at the bottom the admin.. With a perm (cmi.tabsort.[number] or cmi.tabsort.[group]) you can set which rank will show at the top (first) and what rank will follow._
---
### Basic explanation as to why this feature is needed and/or useful to have: ###
_This feature can help players to see Staff members faster, see which rank is highest and in general, the tab would look more orderly and clean._

**Comment by CloudeLecaw** [comment]
This is duplicated request #3571 #3479 #2671 #2474 #1480 #931 #653 


U see its asken often but since 2 years, but the dev may have no time or lust to devlop a feature to sort the playername in the list by some system. You think this will change now ;).?
I think... No ^°^
